sys-process/tiptop/tiptop-2.3.1_p20221211.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+GIT_COMMIT="529886d445ec32febad14246245372a8f244b3eb"
+
+DESCRIPTION="top for performance counters"
+HOMEPAGE="https://github.com/FeCastle/tiptop"
+SRC_URI="https://github.com/FeCastle/tiptop/archive/${GIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+SRC_URI+=" http://deb.debian.org/debian/pool/main/t/tiptop/tiptop_$(ver_cut 1-3)-4.debian.tar.xz"
+S="${WORKDIR}/${PN}-${GIT_COMMIT}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	sys-libs/ncurses:=
+	dev-libs/libxml2:2
+	dev-libs/papi
+"
+DEPEND="${RDEPEND}"
```

Comment:
Note that since we're patching .lex, we need to add a BDEPEND on app-alternatives/lex for now, but put a comment above saying we can drop it once the patch is merged.